spacyfishing/entity_fishing_linker.py

```diff
@@ -207,14 +232,15 @@ def prepare_data(text: str, terms: str, entities: list, language: dict, full: bo
         }
 
     def updated_entities(self, doc: Doc, response: list) -> None:
```

Comment:
catch `AttributeError` for each steps when update span because sometimes span is `None` (that's why the tests don't pass)